src/layout/Group/GroupRenderer.tsx

```diff
@@ -47,7 +49,7 @@ export function GroupRenderer({ node }: GroupRendererProps) {
   return (
     <DisplayGroupContainer
       key={node.item.id}
-      groupNode={node}
+      groupNode={node as LayoutNode<HNonRepGroup, 'Group'>}
```

Comment:
Same as the other comments - I would prefer to wrap the whole thing in `if (node.isNonRepGroup())` so that errors can be caught via the type system instead of just silencing any errors by casting the type.

Reply:
👍  what should the default return be in that case? null?